cmd/stolonctl/upgrade.go

```diff
@@ -33,13 +36,38 @@ var (
 
 func init() {
 	stolonctlCmd.AddCommand(upgradeCmd)
+	upgradeCmd.Flags().StringVar(&clusterConfig.PostgresHost, "postgres-host",
+		defaults.PostgresHost, "Hostname for connection to old stolon PostreSQL host")
+	upgradeCmd.Flags().StringVar(&clusterConfig.PostgresPort, "postgres-port",
+		defaults.PostgresPort, "Port for connection to old stolon PostgreSQL host")
+	upgradeCmd.Flags().StringVar(&clusterConfig.PostgresUser, "postgres-user",
+		defaults.PostgresUser, "Username for connection to old stolon PostgreSQL host")
+	upgradeCmd.Flags().StringVar(&clusterConfig.PostgresPassword, "postgres-password",
+		"", "Password for connection to ols stolon PostgreSQL host")
```

Comment:
ols -> old
I find it a bit odd to refer to `old` Posgres instance w/o `new` - is this necessary?
Additionally, I would tuck all postgres-related attributed into a postgres configuration object.